Makefile

```diff
@@ -134,10 +134,11 @@ TEST := $(shell go run ./test/e2e/split/... -chunks $(E2E_TEST_NUM_CHUNKS) -prin
 endif
 E2E_OPTS ?= $(if $(E2E_SEED),-seed '$(E2E_SEED)') $(if $(SKIP), -skip '$(SKIP)') $(if $(TEST),-focus '$(TEST)') $(if $(ARTIFACT_DIR), -output-dir $(ARTIFACT_DIR) -junit-report junit_e2e.xml) -flake-attempts $(E2E_FLAKE_ATTEMPTS) -nodes $(E2E_NODES) -timeout $(E2E_TIMEOUT) -v -randomize-suites -race -trace -progress
 E2E_INSTALL_NS ?= operator-lifecycle-manager
+E2E_CATALOG_NS ?= $(E2E_INSTALL_NS)
```

Comment:
Setting this to the same value as the OLM installation namespace is intention. This seemed like the best approach to reduce the amount of variables that needed to be overridden between the e2e and e2e-local targets. For upstream environments, you may only need to run `make e2e E2E_CATALOG_NS=olm` for clusters that were already created, and for the e2e-local target, it should be a no-op but there's no integration tests that can easily test all these configuration permutations.